Templates/HttpTrigger/index.js

```diff
@@ -1,10 +1,17 @@
-﻿module.exports = function (req, context) {
+﻿module.exports = function (context, req) {
     context.log('Node.js HTTP trigger function processed a request. Name=' + req.query.name);
 
```

Comment:
We should change this to URI, because if the query string is not present then user will see "undefined" the logs